functionary/ui/templates/partials/table.html

```diff
@@ -1,2 +1,25 @@
 {% load django_tables2 %}
-{% render_table table 'django_tables2/bootstrap5.html' %}
+{% load widget_tweaks %}
+{% if filter %}
+    <form action="" method="get" class="form row">
```

Comment:
Let's put this stuff in its own partial (templates/partials/filter.html).  Then here you would just include that.